app/views/users/index.html.erb

```diff
@@ -1,17 +1,3 @@
-<div class="users-body">
 <% @users.each do |user| %>
-  <a href='/users/<%= user.id %>' class='user-link'>
-    <div class=user-image></div>
-    <div class=user-details>
-      <div class="user-username"><%= user.name %></div>
-      <div class="user-posts-count">number of posts: 
-        <% if user.posts_counter.to_i > 0 %>
-          <%= user.posts_counter %>
-        <% else %>
-          0
-        <% end %>
-      </div>
-    </div>
-  </a>
+<%= render partial: 'partials/user_partial', locals: { user: user } %>
```

Comment:
- [ ] Great job working on the views for users but I think it would be great if you make you layout the same as the given wireframe no styles at this moment but the wireframe it should be the same so kindly work on it to have it similar to the given one 🔥❤🚀
![image](https://user-images.githubusercontent.com/58553711/231930831-4b5a98e7-4cef-4224-ade0-f0ad2963ba3c.png)

- Yours
![image](https://user-images.githubusercontent.com/58553711/231930590-2b736326-424a-41a1-8290-aa121126711d.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/231930689-457b3172-fb35-4f04-a53f-e130747179e5.png)

Reply:
@Reem-lab The entry created at the console fixed